app/components/RegisterSingleTree/MapMarking.js

```diff
@@ -372,7 +379,8 @@ const MapMarking = ({ updateScreenState, inventoryState, resetRouteStack }) => {
           if (IS_ANDROID) {
             updateCurrentPosition();
           } else {
-            Linking.openURL('app-settings:');
+            Linking.openURL('App-Prefs:LOCATION_SERVICES');
```

Comment:
I have not yet tested on Android, but on an iPhone 11 emulator this leads a native crash of the emulator:

<img width="996" alt="Bildschirmfoto 2021-03-08 um 15 34 02" src="https://user-images.githubusercontent.com/1532418/110335038-babb9b80-8023-11eb-9b44-c5f2c023efdc.png">

Reply:
Also an interesting source about this issue: https://developer.apple.com/forums/thread/108866 
Quote: _"All such prefs: and similar URL schemes are private API. Your app will be rejected, even if you did find something that happened to work this week."_ :-)

Reply:
Please let's just revert it to `app-settings:`.

Reply:
sure, if that works.